demos/python_demos/object_detection_demo/object_detection_demo.py

```diff
@@ -0,0 +1,296 @@
+                    threshold=args.prob_threshold, keep_aspect_ratio=args.keep_aspect_ratio)
+    elif architecture_type == 'faceboxes':
+        return FaceBoxes(ie, args.model, log, batch_size=1, threshold=args.prob_threshold)
+    elif architecture_type == 'centernet':
+        return CenterNet(ie, args.model, log, batch_size=1, labels=args.labels, threshold=args.prob_threshold)
+    elif architecture_type == 'retina':
+        return RetinaFace(ie, args.model, log,  batch_size=1, threshold=args.prob_threshold)
+
+    log.error('No such model type as "{}". See "--architecture_type" option for details.'.format(architecture_type))
+    sys.exit(1)
+
+
+def put_highlighted_text(frame, message, position, font_face, font_scale, color, thickness):
+    cv2.putText(frame, message, position, font_face, font_scale, (255, 255, 255), thickness + 1)  # white border
+    cv2.putText(frame, message, position, font_face, font_scale, color, thickness)
+
+
+def get_plugin_configs(device, num_streams, num_threads):
+    config_user_specified = {}
+
+    devices_nstreams = {}
+    if num_streams:
+        devices_nstreams = {device: num_streams for device in ['CPU', 'GPU'] if device in device} \
+            if num_streams.isdigit() \
+            else dict(device.split(':', 1) for device in num_streams.split(','))
+
+    if 'CPU' in device:
+        if num_threads is not None:
+            config_user_specified['CPU_THREADS_NUM'] = str(num_threads)
+        if 'CPU' in devices_nstreams:
+            config_user_specified['CPU_THROUGHPUT_STREAMS'] = devices_nstreams['CPU'] \
+                if int(devices_nstreams['CPU']) > 0 \
+                else 'CPU_THROUGHPUT_AUTO'
+
+    if 'GPU' in device:
+        if 'GPU' in devices_nstreams:
+            config_user_specified['GPU_THROUGHPUT_STREAMS'] = devices_nstreams['GPU'] \
+                if int(devices_nstreams['GPU']) > 0 \
+                else 'GPU_THROUGHPUT_AUTO'
+
+    return config_user_specified
+
+
+def draw_detections(frame, detections, palette, labels, threshold, draw_landmarks=False):
+    size = frame.shape[:2]
+    for detection in detections:
+        if detection.score > threshold:
+            xmin = max(int(detection.xmin), 0)
+            ymin = max(int(detection.ymin), 0)
+            xmax = min(int(detection.xmax), size[1])
+            ymax = min(int(detection.ymax), size[0])
+            class_id = int(detection.id)
+            color = palette[class_id]
+            det_label = labels[class_id] if labels and len(labels) >= class_id else '#{}'.format(class_id)
+            cv2.rectangle(frame, (xmin, ymin), (xmax, ymax), color, 2)
+            cv2.putText(frame, '{} {:.1%}'.format(det_label, detection.score),
+                        (xmin, ymin - 7), cv2.FONT_HERSHEY_COMPLEX, 0.6, color, 1)
+            if draw_landmarks:
+                for landmark in detection.landmarks:
+                    cv2.circle(frame, landmark, 2, (0, 255, 255), 2)
+    return frame
```

Comment:
What's about mask detection ? Will we show result on label/rect ?

Reply:
Yes, look [here](https://github.com/openvinotoolkit/open_model_zoo/pull/1744/files#diff-007e6f32523245c3500b6e3679a99d289e1d701350ac23383cec80e55801df32R38) and [here](https://github.com/openvinotoolkit/open_model_zoo/pull/1744/files#diff-007e6f32523245c3500b6e3679a99d289e1d701350ac23383cec80e55801df32R219) (the last one could be refactored)